src/main/webapp/assets/style.css

```diff
@@ -101,10 +106,15 @@ header {
  */
 
 #map {
+  background: var(--grey);
   height: 500px;
   width: 100%;
 }
```

Comment:
On the mobile version, the map doesn't look proportional as a rectangle. Do you think making it into a square would look better for smaller screens so users know there's more stuff at the bottom to scroll to?

Reply:
I agree! I've added styling for mobile to change it to only take up about half of the user's screen-height, so that they can see where the posts begin